versioned_docs/version-22.04/installation/web-and-post-installation.md

```diff
@@ -50,7 +50,7 @@ Then click on **Next**.
 
 Provide the information needed to connect to the database instance.
 
-- **Database Host Address**: If you are using a local database, leave the field blank (the default value is **localhost**). Otherwise, enter the IP address of your remote database.
+- **Database Host Address**: If you are using a local database, leave the field blank (the default value is **localhost**), or, if you are using Debian, enter 127.0.0.1. Otherwise, enter the IP address of your remote database.
```

Comment:
We fixed this in Broker, not necessary to put 127.0.0.1, localhost works correctly